cli_tools/gce_export/main.go

```diff
@@ -69,99 +91,247 @@ func splitLicenses(input string) []string {
 	return ls
 }
 
-func main() {
-	flag.Parse()
+type buffer struct {
```

Comment:
It implements "writer" and behaves as a writer in this program, so I suggest to call it bufferedWriter